frameworks/elastic/src/test/java/com/mesosphere/sdk/elastic/scheduler/ServiceTest.java

```diff
@@ -68,6 +68,7 @@ private ServiceTestRunner getDefaultRunner() {
                         "ZONE", "us-east-1a")
                 .setBuildTemplateParams(
                         "elastic-version", "1.2.3",
+                        "elastic-statsd-version", "1.2.3",
```

Comment:
would it make sense to specify a different string here ... `7.8.9`?

Reply:
They should be the same value, no?

Reply:
I don't think it matters in terms of tests, but maybe it makes sense setting real-looking values like:
```
elastic-version: 6.1.3
elastic-statsd-version: 6.1.3.0
```

I'll push a change for this.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2078/commits/8f5f500cef864e4051e5d48fa214cffe6dd510e8